librz/analysis/analysis.c

```diff
@@ -85,6 +85,11 @@ RZ_API RzAnalysis *rz_analysis_new(void) {
 		free(analysis);
 		return NULL;
 	}
+	analysis->esilinterstate = RZ_NEW0(RzAnalysisEsilInterState);
+	if (!analysis->esilinterstate) {
+		free(analysis);
+		return NULL;
+	}
```

Comment:
probably this should be converted to a `goto fail`

Reply:
It's somewhat temporary until we get rid of ESIL, so it's probably unnecessary.

Reply:
true